docs/10-action-optimize-media.md

```diff
@@ -0,0 +1,59 @@
+# Add an action to optimize media
```

Comment:
can you also add a link in readme.md to this documentation plz

Reply:
a link to the summary : `ekino/tiny-png-sonata-media-bundle/docs/00-docs.md` is better, nop ?

Reply:
yeah if you want